example/lib/collection.js

```diff
@@ -0,0 +1,29 @@
+var mongodb = require('mongodb');
```

Comment:
Yay promises! Though I should probably publish that to npm...

I will point out that you aren't handling the possible error when connecting to mongo, which is the most likely place for an error to occur. (My rhmap-mongodb automatically catches that, and even retries automatically, btw :) )